### PR TITLE
Optional safeTxGas, gas update

### PR DESCRIPTION
**Pull request by t0mcr8se**
## Resolves
Error GS010, related to gas price and safeTxGas estimation

## How this PR fixes it
Brings SAFE_TX_GAS_OPTIONAL feature from latest version of front-end and updates gas price to 11gwei

### CODE REVIEW

File: src/config/networks/fuse.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ import { fuseBalancesHandler } from 'src/logic/safe/api/fetchTokenCurrenciesBala
 const baseConfig: EnvironmentSettings = {
   clientGatewayUrl: 'https://safe-service.fuse.io/cgw/v1',
   txServiceUrl: 'https://safe-service.fuse.io/txs/api/v1',
-  gasPrice: 1e9, // 1 Gwei TODO: add gasPriceOracles
+  gasPrice: 11e9, // 1 Gwei TODO: add gasPriceOracles
```
Comment (leonprou) [inline src/config/networks/fuse.ts]: it's not working with 10 gwei?

Comment (t0mcr8se) [reply src/config/networks/fuse.ts]: It works, but I see that metamask recommends 11 gwei for most fuse transactions

Comment (leonprou) [reply src/config/networks/fuse.ts]: 10 gwei should be enough.
I saw some comments from the safe team, are they relevant to the PR?

Comment (t0mcr8se) [reply src/config/networks/fuse.ts]: Yes, it's about making the safeTxGas = 0

**Review by leonprou — APPROVED** [review]

**Pull request merged**